### DISCUSSION TITLE
File Icons Usage Policy

### DISCUSSION THREAD

**Discussion by 0x5bfa** [0x7c13/Notepads#1079] [General]
This repo licensed under the MIT License.

_but I felt it is polite to ask._

can i use all file icons in other repos?

**Comment by 0x7c13** [comment]
Code is 100% MIT, but for the icons, I am not that sure. I guess it should also follow MIT umbrella. Just keep in mind that I bought the original set of file icons from third party website (The license for the original ones is allowing me to use for free even for commercial purposes). There are also icons made by @mah-qurashy based on the same design and credits should go for him as well.

So, in general, since it is MIT, you can use it basically whatever you want at this point.

**Reply by 0x5bfa** [reply]
thanks. i will add credits you two.
